lib/OperationHandler.cpp

```diff
@@ -477,9 +507,10 @@ ScanResponse OperationHandler::HandleScanRequest(const ScanRequest& scanRequest)
     return m_serializedRunner.RunAndWait([&] { return HandleScanRequestSerialized(scanRequest); });
 }
 
-void OperationHandler::DrainClientNotifications()
+void OperationHandler::DrainWorkqueues()
 {
-    // Wait for a task doing nothing: this ensure all previous notification have been processed
+    // Wait for a task doing nothing: this ensure all previous tasks have been processed
+    m_serializedRunner.RunAndWait([] { return; });
```

Comment:
Maybe put a trace/log in here so we know when this occurs?

Reply:
This is a test code helper, it isn't used (and shouldn't be used) for production code path.
(see the comment on the declaration, in the header)

Reply:
Got it, I hadn't noticed that, thanks!